translate/convert/json2po.py

```diff
@@ -91,24 +92,33 @@ def convert_unit(self, input_unit, commenttype):
         return output_unit
 
 
-def convertjson(input_file, output_file, template_file, pot=False,
-                duplicatestyle="msgctxt", dialect="default", filter=None):
+def convertjson(
+    input_file,
+    output_file,
+    template_file,
+    pot=False,
+    duplicatestyle="msgctxt",
+    dialect="default",
+    filter=None,
+):
     """Reads in *input_file* using jsonl10n, converts using :class:`json2po`,
     writes to *output_file*.
     """
     from translate.storage import jsonl10n
+
     if filter is not None:
-        filter = filter.split(',')
+        filter = filter.split(",")
     input_store = jsonl10n.JsonFile(input_file, filter=filter)
     convertor = json2po()
     if template_file is None:
-        output_store = convertor.convert_store(input_store,
-                                               duplicatestyle=duplicatestyle)
+        output_store = convertor.convert_store(
+            input_store, duplicatestyle=duplicatestyle
+        )
     else:
         template_store = jsonl10n.JsonFile(template_file)
-        output_store = convertor.merge_store(template_store, input_store,
-                                             blankmsgstr=pot,
-                                             duplicatestyle=duplicatestyle)
+        output_store = convertor.merge_store(
+            template_store, input_store, blankmsgstr=pot, duplicatestyle=duplicatestyle
```

Comment:
nit: i would put 1 per line - at least for `kwargs`

i much prefer the lesser indent tho!

Reply:
Black puts that into one line if it fits and unless there is a trailing comma.

Reply:
> these are generated and not configurable

i guess thats a nit for the linter in that case - in terms of readability i think its better to see the args/kwargs per-line (mostly at least - but def where they are mixed)

i havent had a chance to look at black as i mostly have avoided auto-linting like that, but it would be great if it were configurable in this kind of way